zombies.js

```diff
@@ -240,8 +347,10 @@
  * @param {number} strength         The zombie's strength.
  * @param {number} speed            The zombie's speed.
  */
-
-
+function FastZombie(health, strength, speed) {
+  Zombie.call(this, health, strength, speed);
+}
+FastZombie.prototype = Object.create(Zombie.prototype);
```

Comment:
For this and all the other zombie types below, remember to add the constructor like how you did for Weapon and Item.